rating.cpp

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <conio.h>
+int main(int newGameRes){
+	newGameRes=0;
+	int textArr[1];
```

Comment:
Please use a constant to define number of algorithm.
Also please keep in mind, that there is a situation when human player plays against human player.